### DISCUSSION TITLE
Vizualize TreeBusinessRuleExceptions in response dialog

### DISCUSSION THREAD

**Discussion by melton-jason** [specify/specify7#5711] [Feature Requests]
Currently, TreeBusinessRuleExceptions are all very similar in structure to each other. It would be great to leverage this and design a visual representation of the tree nodes' data from the backend's Json response. Even as something as simple as a table containing the record's id and fullname would greatly increase readability to non-technical users. 

While not a TreeBusinessRuleException, the structure within this Assertion Error is similar. 
![spTreeError](https://user-images.githubusercontent.com/64045831/212233963-03c1b3d6-c760-4984-a2ed-148e4792ff57.png)

A visualization for each of the nodes and basic information about their relationships would also be more compact than formatted Json.

**Comment by maxpatiiuk** [comment]
Some ideas:
A table like in the permissions denied error dialog.
Or a tree made out of nested (optionally collapsible) lists (like in the permissions preview in the security panel for a user account)
